spec/mailers/prior_authority/feedback_messages/part_granted_feedback_spec.rb

```diff
@@ -45,6 +45,8 @@
         defendant_name: 'Abe Abrahams',
         application_total: '£300.00',
         part_grant_total: '£150.00',
+        service_required: 'Pathologist report',
+        service_provider_details: 'ABC DEF<br>ABC, HIJ, SW1 1AA',
```

Comment:
I'm not _sure_ this is going to work. The [documentation](https://www.notifications.service.gov.uk/using-notify/formatting) says it uses markdown formatting so it might be that the <br> will be rendered as a literal string. But I may be way off the mark. Do we pass HTML tags to Notify anywhere else?